README.md

```diff
@@ -1 +1,54 @@
-# proto-schema-registry
\ No newline at end of file
+# proto-schema-registry
+
+[![Action](https://github.com/syncromatics/proto-schema-registry/workflows/build/badge.svg)](https://github.com/syncromatics/proto-schema-registry/workflows/build/badge.svg)
+[![Go Report Card](https://goreportcard.com/badge/github.com/syncromatics/proto-schema-registry)](https://goreportcard.com/report/github.com/syncromatics/proto-schema-registry)
+
+Proto Schema Registry provides a service to store proto schemas in kafka. It also supports breaking change detection and versioning.
+Protobuf generated code is both backwards and forward compatible but this service provides a means for other services that don't have
+the generated protobuf objects the ability to decode and display messages.
+
+## Getting Started
+
+### Installing
+
+#### Docker
+```docker run -e KAFKA_BROKER=broker:9092 -e PORT=443 syncromatics/proto-schema-registry:v0.7.1```
+
+| Environment Variable      | Description                                                                                    |
+|---------------------------|------------------------------------------------------------------------------------------------|
+| KAFKA_BROKER              | A seed kafka broker used to discover the cluster that schema will be stored in. (required)     |
+| PORT                      | The port the grpc service will be hosted at. (required)                                        |
+| REPLICATION_FACTOR        | The replication factor of the schema topic. (defaults to 3)                                    |
+| SECONDS_TO_WAIT_FOR_KAFKA | The seconds to wait for kakfa to be available before the service panics (defaults to 30).      |
+
+### Usage
+
+Check out the [Protobuf Definition](docs/protos/proto/schema/registry/v1/registry_api.proto).
+
+Once the service is up and running use the `RegisterSchema` grpc method to register a schema. The schema must be gzipped in the message.
```

Comment:
This might be better as a list than a paragraph, can you list all three services along with use case?